### DISCUSSION TITLE
Function this.realValueWithoutHarmonic is not getting called

### DISCUSSION THREAD

**Discussion by Tails** [CoderLine/alphaTab#735] [General]
https://github.com/CoderLine/alphaTab/blob/c0d8156574c2ff18d12891ca9d700e37da70c4e6/src/model/Note.ts#L507

The function here is not getting called. Is this intended? Or is it some auto-calling feature of Typescript?

**Comment by kyledecot** [comment]
https://github.com/CoderLine/alphaTab/blob/c0d8156574c2ff18d12891ca9d700e37da70c4e6/src/model/Note.ts#L518

@Tails Since this uses the `get` keyword it doesn't need to be explicitly "called" using parenthesis.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get
